miso-web/src/main/webapp/pages/editSequencerPartitionContainer.jsp

```diff
@@ -108,7 +108,7 @@
 
   <c:if test="${container.id != 0}">
     <miso:qcs id="list_qcs" item="${container}"/>
-    <miso:list-section id="list_partition" name="${container.model.platformType.partitionName}" target="partition" items="${containerPartitions}" config="{ 'platformType' : '${container.model.platformType.name()}', 'showContainer' : false }"/>
+    <miso:list-section id="list_partition" name="${container.model.platformType.partitionName}" target="partition" items="${containerPartitions}" config="{ 'platformType' : '${container.model.platformType.name()}', 'showContainer' : false , 'showPool' : true}"/>
```

Comment:
Can change name to use `pluralPartitionName` instead

Reply:
Should this also be done to the table on the edit Pool page?

Reply:
Oh yeah, there too!